aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -561,7 +562,27 @@ In the following *appsettings.json* example:
 }
 ```
 
-An alternative to using **Path** and **Password** for any certificate node is to specify the certificate using certificate store fields. For example, the **Certificates** > **Default** certificate can be specified as:
+Schema notes:
+
+* Endpoints names are case-insensitive. For example, `HTTPS` and `Https` are valid.
+* The `Url` parameter is required for each endpoint. The format for this parameter is the same as the top-level `Urls` configuration parameter except that it's limited to a single value.
```

Comment:
Can you link back to the Urls parameter so people can easily see the format? If not, duplicate the format here.